### PR TITLE
Order layout

### PR DESCRIPTION
**Pull request by fsosap**
This pull request has the modifications over the views and the order controller methods related with the feedback given to the  user about his or her orders

### CODE REVIEW

File: resources/lang/en/order.php [hunk 1]
```diff
@@ -27,10 +27,22 @@
...
+            'order'=>'Order# ',
+            'orderDate'=>'Order Date: ',
+            'orderSummary'=>'Order Summary: ',
+            'paymentMethod'=>'Payment Method: ',
+            'product'=>'PRODUCT',
+            'quantity'=>'QUANTITY',
             'status'=>'Status: ',
+            'subtotal'=>'SUBTOTAL',
             'total'=>'Total: ',
+            'taxes'=>'Taxes',
```
Comment (ManuelG28) [inline resources/lang/en/order.php:44]: Ponga eso en orden alfabético pana @fsosap xd

File: resources/views/order/details.blade.php [hunk 2]
```diff
@@ -1,105 +1,96 @@
...
+                                                    {{ $data["user"]->getAddress() }}<br>
+                                                    <abbr title="Phone">{{__('order.views.details.email')}}</abbr> {{ $data["user"]->getEmail() }}
+                                                </address>
+                                            </div>
+                                        </div>
+                                        <div class="row">
+                                            <div class="col-xs-6">
+                                                <address>
+                                                    <strong>{{__('order.views.details.paymentMethod')}}</strong><br>
+                                                    Visa ending **** 1234<br>
```
Comment (ManuelG28) [inline resources/views/order/details.blade.php:36]: Usa lang papi bello, no puedes pegar texto ahí @fsosap

**Review by ManuelG28 — COMMENTED** [review]
tqm

File: resources/lang/en/order.php [hunk 3]
```diff
@@ -40,9 +40,9 @@
             'quantity'=>'QUANTITY',
             'status'=>'Status: ',
             'subtotal'=>'SUBTOTAL',
-            'total'=>'Total: ',
             'taxes'=>'Taxes',
-
+            'total'=>'Total: ',
+            'visa'=>'VISA ending: **** 1234',
```
Comment (sgilz) [inline resources/lang/en/order.php]: You should only insert text into the lang section, not special characters or anything that does not need a translation.

**Review by sgilz — APPROVED** [review]

**Pull request merged**